src/Illuminate/Foundation/Testing/DatabaseMigrations.php

```diff
@@ -10,9 +10,13 @@
     public function runDatabaseMigrations()
     {
         $this->artisan('migrate');
+    }
 
-        $this->beforeApplicationDestroyed(function () {
-            $this->artisan('migrate:rollback');
-        });
+    /**
+     * @after
+     */
+    public function runDatabaseMigrateRefresh()
+    {
+        $this->artisan('migrate:refresh');
```

Comment:
I fully agree with @crynobone. In `after` you should always reset the state as it was before the tests ran, not prepare for the next one.